src/python.gen.ts

```diff
@@ -381,6 +393,7 @@ ${this.hooks.join('\n')}
 
     return this.methodSignature(indent, method)
       + this.summary(bump, method.summary)
+      + (method.bodyParams.length > 0 ? this.bodyParamsTypeAssertions(bump, method.bodyParams) : '')
```

Comment:
Seems this decision could be made in `bodyParamsTypeAssertions` instead so this becomes just a plain:
```typescript
+ this.bodyParamsTypeAssertions(bump, method.bodyParams)
```